ci_framework/playbooks/06-deploy-edpm.yml

```diff
@@ -49,10 +49,61 @@
             name: libvirt_manager
             tasks_from: deploy_edpm_compute.yml
 
+        - name: Disable discover_hosts when deploying HCI phase1
+          when: cifmw_edpm_deploy_hci | default(false) | bool
+          ansible.builtin.set_fact:
+            cifmw_edpm_deploy_skip_nova_discover_hosts: true
+
         - name: Deploy EDPM
           ansible.builtin.import_role:
             name: edpm_deploy
 
+- name: Deploy ceph deploy playbook
+  ansible.builtin.import_playbook: ceph.yml
+  when: cifmw_edpm_deploy_hci | default(false) | bool
```

Comment:
You could add vars here so that you don't need to change the defaults in ceph.yaml

```
vars:
    storage_network_range: 172.18.0.0/24
    storage_mgmt_network_range: 172.20.0.0/24
```

Reply:
ack - I will try without nw isolation for now, due to: https://issues.redhat.com/browse/OSPCIX-73